.travis.yml

```diff
@@ -69,18 +66,33 @@ before_install:
 install: 
   - cython --version
   - python --version
+
 before_script:
   - chmod +x build.sh 
 
 script: ./build.sh  
 
-after_failure:
+before_deploy:
   - cd $TRAVIS_BUILD_DIR/build
-  - |
-    echo "config.log: "
-    cat config.log
-    for i in $(find reports -type f); do
-      echo "###########################"
-      echo "File: $i"
-      cat $i
-    done
+  - tar -zcvf configlog.tar.gz ./*.log
+  - tar -zcvf reports.tar.gz ./reports 
+  - mkdir -p $TRAVIS_BUILD_DIR/build/artefacts_upload
+  - cp configlog.tar.gz reports.tar.gz $TRAVIS_BUILD_DIR/build/artefacts_upload
```

Comment:
I almost overlooked the `reports.tar.gz` on this line. For better readability, I would suggest one `cp` command per file. And I guess we could use `mv`, since the archives are not used anywhere else.